app/models/micropost.rb

```diff
@@ -1,8 +1,16 @@
 class Micropost < ApplicationRecord
   belongs_to :user
   has_one_attached :image
+
   scope :newest, ->{order created_at: :desc}
   scope :by_user_id, ->(id){where user_id: id}
+  scope :feed_for, lambda {|id|
+    following_ids = "SELECT followed_id FROM relationships
+                     WHERE follower_id = :user_id"
+    where "user_id IN (#{following_ids}) OR user_id = :user_id",
+          user_id: id
```

Comment:
```suggestion
    following_ids = "SELECT followed_id FROM relationships
                     WHERE follower_id = :user_id" << id
                     
    where "user_id IN (#{following_ids}) "
```
dồn 2 filed đi em, ko gì làm or chi mệt, đều tìm theo user id mà

Reply:
`following_ids` là string thôi mà chị